client/src/components/fieldRadioBtnList/index.js

```diff
@@ -10,11 +9,19 @@ const FieldRadioBtnList = ({ options, name, value, onChange, onBlur, vertical =
           {options.map((o, idx) => {
             const isChecked = o.value === value;
             return (
-              <Button key={idx} color={isChecked ? 'secondary' : 'light' }
-                      onClick={() => {
-                        onChange(name, o.value);
-                      }}>
-                {o.label}</Button>
+              <div key={idx} color={isChecked ? 'secondary' : 'light' }>
```

Comment:
I feel like this should be an additional component FieldRadioList instead of FieldRadioBtnList 

<img width="251" alt="screen shot 2018-05-05 at 8 00 30 am" src="https://user-images.githubusercontent.com/2195815/39654920-71fab6f4-503a-11e8-8400-5d40118ba817.png"

Reply:
I've fixed this now at https://github.com/CodeforAustralia/cese-snippet/tree/master/client/src/components/fieldRadioList